dist/mparticle.js

```diff
@@ -1442,7 +1442,7 @@ var mParticle = (function () {
       var upload = {
         source_request_id: mpInstance._Helpers.generateUniqueId(),
         mpid: mpid,
-        timestamp_unixtime_ms: new Date().getTime(),
+        timestamp_unixtime_ms: mpInstance._Store.SDKConfig.omitBatchTimestamp ? null : new Date().getTime(),
```

Comment:
Please remove this file from the commit. We build this during our release process.